controllers/topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const {fetchTopics} = require('../models/topics.models')
+
+module.exports.getTopics = (request, response) => {
+    fetchTopics().then((topics) => {
+        response.status(200).send({topics})
+    })
```

Comment:
FYI, just doing `exports.getTopics` will work fine here.

`getTopics` and `fetchTopics` are well named functions here, well done!

Sending the response back on a key of `topics` again is a good naming convention as it represents the data we're responding with perfectly, i.e. a bunch of topics, well done!